.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,198 @@
+
+        # Number of matrix instances
+        TOTAL_TEST_GROUPS: 3
+
+      services:
+        clickhouse:
+          image: yandex/clickhouse-server:19.11
+          options: --ulimit nofile=262144:262144
+          ports:
+            - 9000:9000
+
+        snuba:
+          image: getsentry/snuba
+          env:
+            SNUBA_SETTINGS: test
+            REDIS_HOST: redis
+            CLICKHOUSE_HOST: clickhouse
+            CLICKHOUSE_PORT: 9000
+          ports:
+            - 1218:1218
+
+        redis:
+          image: redis
+          ports:
+            - 6379:6379
+
+        memcached:
+          image: memcached
+          ports:
+            - 11211:11211
+
+        postgres:
+          image: postgres:10.8
+          env:
+            POSTGRES_USER: postgres
+          ports:
+            # Maps tcp port 5432 on service container to the host
+            - 5432:5432
+          # needed because the postgres container does not provide a healthcheck
+          options: >-
+            --health-cmd pg_isready
+            --health-interval 10s
+            --health-timeout 5s
+            --health-retries 5
+
+        zookeeper:
+          image: confluentinc/cp-zookeeper:4.1.0
+          env:
+            ZOOKEEPER_CLIENT_PORT: 2181
+
+        kafka:
+          image: confluentinc/cp-kafka:5.1.2
+          env:
+            KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+            KAFKA_LISTENERS: INTERNAL://0.0.0.0:9093,EXTERNAL://0.0.0.0:9092
+            KAFKA_ADVERTISED_LISTENERS: INTERNAL://kafka:9093,EXTERNAL://kafka:9092
+            KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: INTERNAL:PLAINTEXT,EXTERNAL:PLAINTEXT
+            KAFKA_INTER_BROKER_LISTENER_NAME: INTERNAL
+            KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+
+      steps:
+        # Checkout codebase
+        - uses: actions/checkout@v2
+
+        # See https://github.com/actions/cache/blob/master/examples.md#node---yarn for example
+        - name: Get yarn cache directory path
+          id: yarn-cache-dir-path
+          run: echo "::set-output name=dir::$(yarn cache dir)"
+
+        # Use nvmrc so we don't have yet another place node version is defined
+        - name: Get node version from .nvmrc
+          id: nvmrc
+          run: echo "::set-output name=version::$(cat .nvmrc)"
+
+        # yarn cache
+        - uses: actions/cache@v1
+          id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+          with:
+            path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+            key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+            restore-keys: |
+              ${{ runner.os }}-yarn-
+
+        # Setup node
+        - uses: actions/setup-node@v1
+          with:
+            node-version: ${{ steps.nvmrc.outputs.version }}
+
+        # setup python
+        - name: Set up Python 2.7
+          uses: actions/setup-python@v1
+          with:
+            python-version: 2.7
```

Comment:
do you know what minor version of python 2.7 is installed?

Reply:
https://github.com/actions/python-versions/releases?after=3.5.4-20200429.23

Reply:
Oh, it seems it's possible to specify minor. Can you pin to 2.7.16 and add a comment to keep in-sync with the `.python-version` file? I couldn't quickly find a way to read from that file, and this will rarely be updated, so it's fine to dupe imo.

Reply:
You could add a step to cat the python version and then store it in a env var. See https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
Oh billy already did something like this above.

Reply:
Wait @EvanPurkhiser where did you see that?

Reply:
Anyway, right now looks like it's failing with `Version 2.7.16 with arch x64 not found`. I don't think Billy ever pinned 2.7.16

Reply:
https://github.com/getsentry/sentry/pull/18576/files/65b4d1d879f8372341a2054926d05bb4e311172b#diff-a83a4971812cfaf62e21715eea2c1d5bR129-R131

Reply:
But that's for node. Oh, I guess you were talking about something else, it wasn't clear.